src/x/mmap/mmap_other.go

```diff
@@ -81,3 +82,18 @@ func Munmap(b []byte) error {
 
 	return nil
 }
+
+// Free mmapped memory.
+func Free(b []byte) error {
+	return madvise(b, syscall.MADV_DONTNEED)
+}
+
+// This is required because the unix package does not support the madvise system call on OS X.
```

Comment:
Ah interesting, is there other places where this is obvious?

Reply:
Hmm, I'm not sure although we're technically only building for linux in our actual deployments?

Reply:
Although, on second thought... I'll update the comment because the `SYS_MADVISE` constant is defined for all other builds as well in `runtime` so it should work generically. 

https://golang.org/search?q=SYS_MADVISE